src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -618,6 +619,15 @@ public string PythonTemplateFilePath
         /// </summary>
         public bool EnableNodeAutoComplete { get; set; }
 
+        /// <summary>
+        /// PolyCurve normal and direction behavior has been made predictable in Dynamo 3.0 and has therefore changed. 
+        /// This defines whether legacy (pre-3.0) PolyCurve behavior is selected by default.
+        /// This flag can be overridden by individual workspaces that have the EnableLegacyPolyCurveBehavior flag defined.
+        /// Note: For internal use only and will be removed in a future version of Dynamo.
+        /// </summary>
+        [IsObsolete("This property will be removed in a future version of Dynamo.")]
```

Comment:
@mjkkirschner couldn't make this internal nor could I use the `Obsolete` attribute so I used the bespoke one we use in Dynamo instead and updated the summary. PTAL.

Reply:
could you show what happened when you try to use the real Obsolete attribute?

Reply:
It just doesn't serialize.

Reply:
https://stackoverflow.com/questions/331013/obsolete-attribute-causes-property-to-be-ignored-by-xmlserialization